src/libsync/discoveryphase.cpp

```diff
@@ -299,12 +304,14 @@ void DiscoverySingleDirectoryJob::directoryListingIteratedSlot(QString file, con
 {
     //qDebug() << Q_FUNC_INFO << _subPath << file << map.count() << map.keys() << _account->davPath() << _lsColJob->reply()->request().url().path();
     if (!_ignoredFirst) {
-        // First result is the directory itself. Maybe should have a better check for that? FIXME
+        // The first entry is for the folder itself, we should process it differently.
         _ignoredFirst = true;
         if (map.contains("permissions")) {
             emit firstDirectoryPermissions(map.value("permissions"));
         }
-
+        if (true || map.contains("data-fingerprint")) {
```

Comment:
DiscoverySingleDirectoryJob is an internal class to the discovery job, it's not really a public class with an exported interface. Its role is to abstract the protocol part of the discovery.  In that sens, the name of the properties are part of the protocol and should be IMHO kept inside this class.